Makefile.am

```diff
@@ -617,6 +618,14 @@ builtin_modules += android_wakelock
 builtin_sources += plugins/android-wakelock.c
 endif
 
+if ACCOUNTSSETTINGS
```

Comment:
AccountsService is a freedesktop project [1], with a standard DBus interface, and it is used by different distributions, including Ubuntu and Fedora.

[1] https://www.freedesktop.org/wiki/Software/AccountsService/